llvm/test/tools/repo2obj/extern_weak_references.json

```diff
@@ -21,31 +21,29 @@
     {
       // generation 1
```

Comment:
Again, should change `generation` to `transaction`?

Reply:
As above